moneta/src/python/customauth/views.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import (authenticate, login, logout)
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render, redirect
+
+from .forms import UserLoginForm
+
+
+@login_required
+def home(request):
+    return render(request, 'home.html')
+
+
+def login_view(request):
+    # if not request.user.is_authenticated:
+    #     return redirect("/")
+    next = request.GET.get('next')
+    form = UserLoginForm(request.POST)
+    print(form.is_valid())
```

Comment:
print??? Why not logging?

Reply:
It was only for checking the work of my function. Currently, I deleted this